### ISSUE TITLE
Update PostCSS dependency

### ISSUE THREAD

**Issue by sambauers** [TehShrike/svelte-preprocess-postcss#8]
I get the following error with `postcss@7.x` installed, which is working fine so far.

```
npm WARN svelte-preprocess-postcss@1.1.0 requires a peer of postcss@6.x but none is installed. You must install peer dependencies yourself.
```

Can the PostCSS dependency be updated to 7.x or is there an issue with that I can help with?

**Comment by TehShrike** [comment]
If you've tested it with postcss 7 and it's all good, then yeah, I'd appreciate a PR that changes the peer dependency to include postcss 6 and 7 👍

**Comment by sambauers** [comment]
See #9 

Not sure if package lock file should be updated too.

**Comment by TehShrike** [comment]
Published as 1.1.1 👍
